org.adempiere.base/src/org/compiere/model/PO.java

```diff
@@ -153,15 +153,26 @@ public PO (Properties ctx)
 
 	/**
 	 *  Create and Load existing Persistent Object
-	 *  @param ID The unique ID of the object
 	 *  @param ctx context
+	 *  @param ID The unique ID of the object
 	 *  @param trxName transaction name
 	 */
 	public PO (Properties ctx, int ID, String trxName)
 	{
 		this (ctx, ID, trxName, null, (String[]) null);
 	}   //  PO
 
+	/**
+	 *  Create and Load existing Persistent Object
+	 *  @param ctx context
+	 *  @param UUID The unique UUID of the object
```

Comment:
Needs to document what's the value for new record ("" or a public constant like UUID_NEW_RECORD)

Reply:
> Needs to document what's the value for new record ("" or a public constant like UUID_NEW_RECORD)

Ready, added PO.UUID_NEW_RECORD and changed test class accordingly